### PR TITLE
add set hiddne api policy

### PR DESCRIPTION
**Pull request by KazuCocoa**
addressed in https://github.com/appium/appium-android-driver/pull/461#issuecomment-435773760

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -264,11 +264,18 @@ class AndroidUiautomator2Driver extends BaseDriver {
...
-    if (await this.adb.getApiLevel() < 21) {
+    const apiLevel = await this.adb.getApiLevel();
+
+    if (apiLevel < 21) {
       logger.errorAndThrow('UIAutomation2 is only supported since Android 5.0 (Lollipop). ' +
         'You could still use other supported backends in order to automate older Android versions.');
     }
 
+    if (apiLevel >= 28) { // Android P
+      logger.warn('Relaxing hidden api policy');
```
Comment (joshuapinter) [inline lib/driver.js:275]: Can we change this to `logger.info`? Two reasons: 

1. It matches the log level further down where you reset the API Policy to the default: 

    ```js
    if (await this.adb.getApiLevel() >= 28) { // Android P
      logger.info('Restoring hidden api policy to the device default configuration');
      await this.adb.setDefaultHiddenApiPolicy();
    }
    ```

2. It's not really a warning if it's always set and not something we can actually fix/remediate.

Plus, it shows up in our console logs, which are warning only and it's noisy: 

image

Comment (mykola-mokhnach) [reply lib/driver.js:275]: @joshuapinter Feel free to push a PR

Comment (joshuapinter) [reply lib/driver.js:275]: Excellent, that's what I wanted to hear.

source

Comment (joshuapinter) [reply lib/driver.js:275]: PR pushed at #396

**Pull request merged**